en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -4,7 +4,7 @@ A RESTful API (Application Program Interface) uses HTTP requests to access and u
 
 Choreo’s low-code editor allows developers to easily design (and then implement) high-quality REST APIs. To explore this capability, let's consider a scenario where an Analyst needs to retrieve the daily count of COVID-19 patients per one million population by country. In this tutorial, you will address this requirement by doing the following:
 
-- Design a REST API that addresses the described requirement, test it in the VS Code Editor, and then commit it so that it is available in the Chore Console.
+- Design a REST API that addresses the described requirement, test it in the web editor, and then commit it so that it is available in the Chore Console.
```

Comment:
Let's have 'Web Editor' in title caps because it is an application.

Reply:
But since it's a general term, should we capitalise it?